### ISSUE TITLE
Show error if backend call fails

### ISSUE THREAD

**Issue by arvandf** [rrebase/knboard#31]
If during login (or probably other parts) backend has an error or is down, UI doesn't show an error. Probably it will be useful if an error is displayed.

**Comment by rrebase** [comment]
Most places should have errors displayed. Is it network errors that are not shown?

**Comment by arvandf** [comment]
Yes, for example, is backend not running and I try to login no error is shown. I am not about error messages that are returned by endpoints.
